app/policies/user_policy.rb

```diff
@@ -0,0 +1,17 @@
+class UserPolicy
+  attr_reader :current_user, :user
+  
+  def initialize(current_user, user)
+    @current_user = current_user
+    @user = user
+  end
+  
```

Comment:
Hello Andrew! The liked? and discover? actions are missing so when I click discover, I get the following error message "NoMethodError at /lala/discover
undefined method `discover?"

Reply:
I fixed the error as you suggested. Thank you Alana!